sessionctx/variable/session.go

```diff
@@ -966,6 +966,12 @@ type SessionVars struct {
 
 	// EnablePaging indicates whether enable paging in coprocessor requests.
 	EnablePaging bool
+
+	// KvExecCounter is created from session.stmtStats to count the number
+	// of SQL executions of the kv layer during the current execution of
+	// the statement. Its life cycle is limited to this execution, and a
+	// new KvExecCounter is always created during each statement execution.
+	KvExecCounter *stmtstats.KvExecCounter
```

Comment:
`KvExecCounter` already in `StatementContext `, why also add in here?

Reply:
It seems that there is no need to maintain an additional pointer here, I have transferred all the use of KvExecCounter to StatementContext.